hash/vsmile_cart.xml

```diff
@@ -165,12 +165,13 @@ Regular game cartridges
 |        | 80-092062(NL)     | Winnie de Poeh - De wonderwereld van Poeh                                                      |
 |   XX   | 80-092062-123(NL) | Winnie de Poeh - De wonderwereld van Poeh (rev. 123, labeled as "My Friends Tigger and Pooh", 52-092062(NL) on back label)
 |        | 80-092063(UK)     | Winnie The Pooh: The Honey Hunt                                                                |
+|   XX   | 80-092063-103(UK) | Winnie The Pooh: The Honey Hunt (52-092063-103(UK)) on back label                              |
```

Comment:
The format isn’t the same between these where “on back label” is inside the parentheses for the NL version, but outside the parentheses for the UK version.

Reply:
Arggggg... Yes, sorry.